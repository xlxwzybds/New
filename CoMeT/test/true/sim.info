{'cmdline': ['../../run-sniper', '-n', '1', '--', './true'],
 'git_revision': '4ef841cd19f7a43fe2b912893f039737fe24b11a',
 'host': 'shailja-OptiPlex-7050',
 'pin_version': 'unknown',
 'rusage': (0.15953399999999998,
            0.034492999999999996,
            33076,
            0,
            0,
            0,
            6678,
            0,
            0,
            0,
            144,
            0,
            0,
            0,
            3560,
            990),
 'snipercmd': ['bash',
               '-c',
               '/home/shailja/temp/sniper_7.3_major_bug_fixing/lib/sniper -c /home/shailja/temp/sniper_7.3_major_bug_fixing/config/base.cfg --general/total_cores=1 --general/output_dir=/home/shailja/temp/sniper_7.3_major_bug_fixing/test/true --config=/home/shailja/temp/sniper_7.3_major_bug_fixing/config/nehalem.cfg --config=/home/shailja/temp/sniper_7.3_major_bug_fixing/config/gainestown.cfg -g --traceinput/stop_with_first_app=true -g --traceinput/restart_apps=false -g --traceinput/stop_with_first_app=false -g --traceinput/enabled=true -g --traceinput/emulate_syscalls=true -g --traceinput/num_apps=1 -g --traceinput/trace_prefix=/tmp/tmpYyUqXg/run_benchmarks'],
 't_elapsed': 3.254065990447998,
 't_start': 1606315186.302144,
 'tracecmds': [['/home/shailja/temp/sniper_7.3_major_bug_fixing/record-trace',
                '-o',
                '/tmp/tmpYyUqXg/run_benchmarks',
                '-e',
                '1',
                '-s',
                '0',
                '-r',
                '1',
                '--follow',
                '--routine-tracing',
                '--',
                './true']],
 'user': 'shailja',
 'vmem': 42524672L}
