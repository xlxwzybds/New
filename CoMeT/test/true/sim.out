                                     | Core 0    
  Instructions                       |      42946
  Cycles                             |     118902
  IPC                                |       0.36
  Time (ns)                          |      44700
  Idle time (ns)                     |        -28
  Idle time (%)                      |      -0.1%
Branch predictor stats               |           
  num correct                        |       9748
  num incorrect                      |       1610
  misprediction rate                 |     14.18%
  mpki                               |      37.49
TLB Summary                          |           
  I-TLB                              |           
    num accesses                     |       5676
    num misses                       |         44
    miss rate                        |      0.78%
    mpki                             |       1.02
  D-TLB                              |           
    num accesses                     |      10263
    num misses                       |         22
    miss rate                        |      0.21%
    mpki                             |       0.51
  L2 TLB                             |           
    num accesses                     |         66
    num misses                       |         66
    miss rate                        |    100.00%
    mpki                             |       1.54
Cache Summary                        |           
  Cache L1-I                         |           
    num cache accesses               |       5676
    num cache misses                 |        459
    miss rate                        |      8.09%
    mpki                             |      10.69
  Cache L1-D                         |           
    num cache accesses               |      10263
    num cache misses                 |        318
    miss rate                        |      3.10%
    mpki                             |       7.40
  Cache L2                           |           
    num cache accesses               |        802
    num cache misses                 |        777
    miss rate                        |     96.88%
    mpki                             |      18.09
  Cache L3                           |           
    num cache accesses               |        802
    num cache misses                 |        777
    miss rate                        |     96.88%
    mpki                             |      18.09
DRAM summary                         |           
  num dram accesses                  |        777
  average dram access latency (ns)   |      79.77
  average dram queueing delay        |      26.34
  average dram bandwidth utilization |     14.00%
Coherency Traffic                    |           
  num loads from dram                |        178
  num loads from dram cache          |          0
  num loads from remote cache        |          0
